libraries/AC_AttitudeControl/AC_PosControl_Sub.h

```diff
@@ -31,6 +31,15 @@ class AC_PosControl_Sub : public AC_PosControl {
     ///     set force_descend to true during landing to allow target to move low enough to slow the motors
     void set_alt_target_from_climb_rate_ff(float climb_rate_cms, float dt, bool force_descend) override;
 
+    /// relax_alt_hold_controllers - set all desired and targets to measured
+    void relax_alt_hold_controllers(float throttle_setting) {
```

Comment:
This definition is redundant/implicit. You can remove it.

Reply:
It's not possible to access it since we have a function with the same name (and the original one is a virtual), this definition is necessary.

Reply:
you are right, but the reason is explained here: https://stackoverflow.com/questions/1896830/why-should-i-use-the-using-keyword-to-access-my-base-class-method

what you have now is fine to me